src/modules/Security.rb

```diff
@@ -64,19 +63,18 @@ def import_modules
     end
 
     def init_settings
-
       # Services to check
       srv_file = Directory.find_data_file("security/services.yml")
-      if srv_file
-        srv_lists = YAML.load_file(srv_file) rescue {}
-      else
-        srv_lists = {}
-      end
+      srv_lists = srv_file ? (begin
+                                YAML.load_file(srv_file)
+                              rescue
+                                {}
+                              end) : {}
```

Comment:
thats matter of taste. I can use your approach. However I prefer the "functional" style returning and assigning the value of if.